src/status_im/data_store/realm/schemas/base/core.cljs

```diff
@@ -1,11 +1,15 @@
 (ns status-im.data-store.realm.schemas.base.core
   (:require [status-im.data-store.realm.schemas.base.v1.core :as v1]
-            [status-im.data-store.realm.schemas.base.v2.core :as v2]))
+            [status-im.data-store.realm.schemas.base.v2.core :as v2]
+            [status-im.data-store.realm.schemas.base.v3.core :as v3]))
 
 ; put schemas ordered by version
 (def schemas [{:schema v1/schema
                :schemaVersion 1
                :migration v1/migration}
               {:schema v2/schema
                :schemaVersion 2
-               :migration v2/migration}])
+               :migration v2/migration}
```

Comment:
Could you align these values? So it looks like:
```
{:a          1
 :foobar 2
```

We had a conversation in Slack a while ago and decided all PRs should have this going forward.

Reply:
Indeed!